apps/style/music/style.scss

```diff
@@ -1 +1,4 @@
 /* This file is currently empty. */
+.blahblahblah {
```

Comment:
to be removed?

Reply:
Hah, yes.  Was trying to figure out what gets loaded when.